app/src/locale/translation.json

```
@@ -36,9 +36,7 @@
     },
     "search_ph": {
         "en": "Search for a hall...",
-
-        "sv-sv": "Sök efter en sal...",
-
+        "sv-sv": "Sök efter ett sal...",
```

```suggestion
        "sv-sv": "Sök efter en sal...",
```

Fortfarande samma typo. Bra om vi fixar det så vi inte glömmer det.